src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -1763,7 +1883,7 @@ public String getContextPath() throws IOException {
         public WebRequest addCrumb(WebRequest req) {
             NameValuePair crumb = new NameValuePair(
                     jenkins.getCrumbIssuer().getDescriptor().getCrumbRequestField(),
-                    jenkins.getCrumbIssuer().getCrumb( null ));
+                    jenkins.getCrumbIssuer().getCrumb((javax.servlet.ServletRequest) null));
```

Comment:
Here and elsewhere, newer cores will have two versions of this method, one for `javax` and one for `jakarta`, so to be forward compatible we have to disambiguate the method call. Here we choose the older import, since this change does have to work on non-Jakarta cores.